modules/upi-removing-hosts-from-a-upi-cluster.adoc

```diff
@@ -74,6 +74,18 @@ openshift-machine-api   controller1   externally provisioned              true
 openshift-machine-api   worker1       deprovisioning                      true             57m
 ----
 
+. Delete the host by running the following command when the `BareMetalHost` state changes to `available`:
++
+[source,terminal]
+----
+$ oc delete bmh -n openshift-machine-api <bmh_name>
+----
++
+[NOTE]
+====
+You can run this step without having to cordon and drain the node. It might take some time for the `BareMetalHost` state to change from `deprovisioning` to `available`. 
```

Comment:
The previous step here was step 2. `Delete the customDeploy specification from the BareMetalHost CR`. We can run the delete without doing the edit of the baremetalhost. If step 2 is skipped the delete command seems like it is taking forever to complete. In the background the node is changing state from deprovisioning to available to finally being deleted. Cordoning and draining the node is a graceful way to mark a node unschedulable and to remove workloads. It's an important step.

Reply:
Good catch. Thanks!